sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1168,6 +1199,7 @@ private async Task RunAsync(CancellationToken cancellationToken)
                     // TODO: should the handler be notified in the processing task instead?  User will be notified
                     // earlier.
 
+                    Logger.StopPartitionProcessingError(partitionId, reason, ex.Message);
```

Comment:
I think the name is misleading here. `StopPartitionProcessing` did not fail. The partition processing task has failed and we are simply grabbing the exception it threw and passing it to the error handler to be processed. I think renaming the method to `PartitionProcessingError` would be more accurate.

Reply:
Also, I don't think that logging the `reason` here would be meaningful. Like I said, this is actually a partition processing error, so including a `reason` wouldn't make much sense here because this parameter is directly related to the `StopPartitionProcessing` activity, and not to the partition processing task.